### ISSUE TITLE
classic midi conections

### ISSUE THREAD

**Issue by ajabadia** [rsta2/pico#4]
Good evening, I am testing your code on my raspberry pi pico and I found it very good.
I am not a programmer so it is difficult for me to understand the code, although I see that it is a clean code.
I would like to ask you if it is possible to add a classic midi in, out and thru (5 pins din connectors) using other uart pins (in addition to the optocouplers, resistors...).
It seemed to me that only uart 0 was initialized, so I guess it could not be done, is that so?
Would it be too complicated to add this functionality?
greetings and thank you very much for your time

Translated with DeepL.com (free version)

**Comment by rsta2** [comment]
The midi_adapter_flex variant supports uart0 and uart1 and an USB MIDI 
interface. Normally it has to be configured by the user for his/her purpose. 
You can direct any received MIDI event from one port to any of the other 
ports. Unfortunately I cannot provide schematics / PCB for the hardware 
things, you have requested. My business is software, not hardware.

**Comment by ajabadia** [comment]
II hadn't noticed the Flex version but I'm going to try it out as soon as I can. Since there is more than one uart port enabled, adding classic MIDI should be straightforward, there are several schematics around the internet for this. I'll see which scheme fits best and if I succeed, I'll share with you how I did it in case you want to leave this option documented in your project.
Best regards and thank you very much
